package.json

```diff
@@ -18,11 +18,19 @@
     },
     "homepage": "https://github.com/NJaryal/ReactApp#readme",
     "dependencies": {
+        "@babel/polyfill": "^7.4.4",
+        "babel-polyfill": "^6.26.0",
```

Comment:
seems not required

Reply:
Required as I was getting regenerator error.

As mentioned in the above comment.
I have only kept babel-polyfill though.

Reply:
yep, sorry, didn't notice it's polyfill, correct